services/workflows-service/src/common/get-file-extension-with-fallback/get-file-extension-with-fallback.ts

```diff
@@ -16,7 +15,6 @@ export const getFileExtensionWithFallback = async ({
   fileName?: string;
 }) => {
   const fileMetadata = await getFileType(file);
-  const mimeType = await getMimeType({ file, fileName });
```

Comment:
im not sure about this deletion 
you can extract mimetype using this function in case fileMetadata?.mime is not defined 

let mimeType = fileMetadata?.mime
if(!mimeType) {
mimeType = await getMimeType({ file, fileName });
}